src/Components/App/App.js

```diff
@@ -1,20 +1,52 @@
 import './App.scss';
-import {Homepage} from '../HomePage/Homepage'
-import Header from '../Header/Header'
-import React, {Component} from 'react'
-import {Route} from 'react-router-dom'
+import {Homepage} from '../HomePage/Homepage';
+import Header from '../Header/Header';
+import { Form } from '../Form/Form';
+import React, {Component} from 'react';
+import {Route} from 'react-router-dom';
 
 class App extends Component {
   constructor() {
-    super()
+    super();
+    this.state = {
+      activities: [],
```

Comment:
We may be able to get rid of this `activities` array in state now that our functionality is more flushed out.  We could potentially create some logic to determine the activities based on which activity array (e.g. `moviesAnswers`) is not empty.

Reply:
I think having the `activities` as a state can be helpful if we decide to render information about the user's chosen activities.

Reply:
I think we could potentially just do any render conditionals based on what Nicole is recommending, but if you have a specific functionality in mind that is cool too.